framework/console/controllers/FixtureController.php

```diff
@@ -415,13 +415,36 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            $relativeName = $this->getFixtureRelativeName($fixture);
+            $foundFixtures[] = $relativeName;
         }
 
         return $foundFixtures;
     }
 
     /**
+     * Calculates $fixture's name relatively to $templatePath.
+     * Basically, strips getFixturePath() and 'Fixture.php' prefix from fixture's full path
+     * @see getFixturePath()
+     * @param $fixture string Full fixture path
+     * @return string Relative fixture name
+     *
+    */
+    private function getFixtureRelativeName($fixture) {
+        // $fixturesPath is normalized to unix format in getFixturesPath()
+        $fixturesPath = $this->getFixturePath();
+        // normalize $fixture to unix format
+        $fixture = str_replace("\\", "/", $fixture);
+        // strip $fixturesPath from $fixture's full path
+        $relativeName = str_replace($fixturesPath . "/", "", $fixture);
+        // get fixtures's directory
+        $relativeDir = dirname($relativeName) === '.' ? '' : dirname($relativeName) . '/';
+        // get fixture name relatively to $fixturesPath
+        $relativeName = $relativeDir . basename($fixture, 'Fixture.php');
+        return $relativeName;
+    }
```

Comment:
it's hard to determine the quality of the logic without a dedicated unittest. It would be nice to have one for a proper understanding.